cmd/minishift/cmd/util/util_test.go

```diff
@@ -32,6 +32,9 @@ func TestIsValidProfileName(t *testing.T) {
 		{"foo_123", true},
 		{"foo@123", false},
 		{"", false},
+		{"a", true},
```

Comment:
We should add `{"1", true}` to test single digit profile names.

Reply:
@LalatenduMohanty Since regex is for word range so testing a digit or alphabet would be same but here we are testing single character (be it digit or alphabet).

Reply:
That's the beauty of unit tests. If we cover all possible combinations of profile name in the unit test then in future eve if we change the regular expression in the function , it will make sure that we do not introduce a regression.